app/helpers/degrees_helper.rb

```diff
@@ -5,7 +5,7 @@ module DegreesHelper
 
   def degree_type_options
     to_enhanced_options(degree_type_data) do |ref_data|
-      synonyms = (ref_data[:synonyms] || []) << ref_data[:abbreviation]
+      synonyms = (([ref_data[:match_synonyms]] + [ref_data[:suggestion_synonyms]] + [ref_data[:abbreviation]]).flatten || [])
```

Comment:
You could try this:
```suggestion
      synonyms = Array(ref_data[:match_synonyms]) + Array(ref_data[:suggestion_synonyms]) + Array(ref_data[:abbreviation])
```

Passing nil should create an empty array:
```ruby
>> Array(nil) #=> []
```